client/asset/btc/btc.go

```diff
@@ -2029,6 +2017,23 @@ func (btc *ExchangeWallet) send(address string, val uint64, feeRate uint64, subt
 	return nil, 0, 0, fmt.Errorf("failed to locate transaction vout")
 }
 
+func (btc *ExchangeWallet) sendRawTransaction(tx *wire.MsgTx) (*chainhash.Hash, error) {
+	if btc.legacyRawFeeLimit {
+		return btc.node.SendRawTransaction(tx, false)
+	}
+	b, err := serializeMsgTx(tx)
+	if err != nil {
+		return nil, err
+	}
+	var txid string
+	err = btc.node.call(methodSendRawTx,
+		anylist{hex.EncodeToString(b), float64(btc.feeRateLimit) * 1e-5}, &txid)
```

Comment:
I just noticed `feeRateLimit` here.  Wallet should not be using `feeRateLimit` since it was only added
to help Core decide if it will work with a given server.  For sendrawtransaction, this should just be something large by default (or omitted).

Reply:
I agree. This is an optional argument, and we just shouldn't specify it here, defaulting to the node's default e.g. `0.1` for bitcoind. `feeratelimit` check is for `FundOrder`.